### PR TITLE
Add planned enhancements into DG

### PR DESCRIPTION
**Pull request by EksdeeeX**
- Added planned enhancements into DG.
- Closes issue:
    - #199 
    - #202 
    - #203 
    - #214

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -857,7 +857,18 @@ testers are expected to do more *exploratory* testing.
 
 ## Appendix: Planned Enhancements
 1. The current `newevent` and `edit` commands allows an event's start time to be after its end time. We plan to only allow an event to be added or successfully edited if its start time is before or the same as its end time, with an error message `Start time must be before end time.` being shown otherwise.
+
 2. The current `linkcontact` command does not recognise an inputted phone number if there are extra spaces around it. For example, `linkcontact 1      12345678` is not allowed. We plan to allow the extra spaces by ignoring extra spaces when splitting the input arguments.
+
+3. upon using the current `linkcontact` command on a contact that is already linked to a specified event, the application returns "contact successfully linked" message in the message display box. We plan to change the display message to "Contact already linked" instead for contacts that are already linked.
```
Comment (glozxi) [inline docs/DeveloperGuide.md:863]: Should the first letter be capitalised?

Comment (glozxi) [inline docs/DeveloperGuide.md:863]: Should the messages be in `code format` and have similar capitalisation to be more consistent?

**Review by glozxi — CHANGES_REQUESTED** [review]
LGTM, just some nits

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (EksdeeeX) [reply docs/DeveloperGuide.md:863]: good suggestion

Comment (EksdeeeX) [reply docs/DeveloperGuide.md:863]: good point

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -860,7 +860,7 @@ testers are expected to do more *exploratory* testing.
 
 2. The current `linkcontact` command does not recognise an inputted phone number if there are extra spaces around it. For example, `linkcontact 1      12345678` is not allowed. We plan to allow the extra spaces by ignoring extra spaces when splitting the input arguments.
 
-3. upon using the current `linkcontact` command on a contact that is already linked to a specified event, the application returns "contact successfully linked" message in the message display box. We plan to change the display message to "Contact already linked" instead for contacts that are already linked.
+3. Upon using the current `linkcontact` command on a contact that is already linked to a specified event, the application returns a 'Successfully linked' message in the message display box. We plan to change the display message to 'Contact already linked' instead for contacts that are already linked.
```
Comment (glozxi) [inline docs/DeveloperGuide.md:863]: I think that the messages should be in`` instead of '' to be consistent with the message in point 1.

Comment (EksdeeeX) [reply docs/DeveloperGuide.md]: fixed!

**Pull request merged**